### DISCUSSION TITLE
I lost the ability to link my files on github with files on local computer (in VisualStudioCode)

### DISCUSSION THREAD

**Discussion by Boggitan** [community/community#62994] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

After RELOAD REQUIRED the Git SCM Integration, I lost the ability to connect my files on the local computer with remote github repositories. I working at VSCode. How can I fix it problem?

**Comment by fabricio-fn** [comment]
try to reconnect your account, also see if the git settings are correct using
git config --global user.name
git config --global user.email

if not use 
git config --global user.name "Your Name"
git config --global user.email "your@email.com"

in your project directory
use the command "git remote -v" to check the configured remote URLs for the repository. see if they are correct.

if nothing works restart everything hahaha

**Reply by Boggitan** [reply]
No!!! It`s so risk! hahaha. But I really don't understand what to do...

**Comment by mehulrajdev** [comment, marked as answer]
use git init

https://www.educative.io/answers/the-fatal-refusing-to-merge-unrelated-histories-git-error
this might be helpful

**Reply by Boggitan** [reply]
Thank`s
